src/io/flutter/preview/PreviewView.java

```diff
@@ -185,7 +185,7 @@ public void onFailure(RenderProblemKind kind) {
               previewArea.clear(PreviewArea.NO_WIDGET_MESSAGE);
               break;
             case NOT_RENDERABLE_WIDGET:
-              previewArea.clear("Selection does not correspond to a renderable widget");
+              previewArea.clear("Selection not a renderable widget");
```

Comment:
Some other alternatives.
"not" => "is not"?
Or maybe just "Not a renderable widget"?

Reply:
I'll change to `is not`